routes/ci.js

```diff
@@ -208,6 +214,11 @@ var execDocker = function(buildId, buildItemId, accessToken) {
               console.error('Can\'t start the container ' + CIBuildContainerName);
               reject(err);
             }
+            // Set timeout for the maximum uptime.
+            setTimeout(function() {
+              var err = new Error('Uptime for ' + CIBuildContainerName + ' is overtime.');
```

Comment:
not sure we need to `reject` - maybe you can just `throw new Error('Uptime for ' + CIBuildContainerName + ' is overtime.');`

Also message should be `CIBuildContainerName + execution has timed out after ' + TIME + 'minutes';`